.env.example

```diff
@@ -0,0 +1,2 @@
+NETWORK=ethereum-mainnet
+TRANSACTION_NAME=test
```

Comment:
can't we use the filename instead of an env variable?

Reply:
I agree that it is more simple.

Reply:
I'm not sure I understand your suggestion @MerlinEgalite

Reply:
forget about that

Reply:
Perhaps shouldn't I have mixed my suggestion with this discussion. I see you upvoted my comment Patrick, did you forget about implementing the suggested change?